sheetsync/sheetsync/sheets.py

```diff
@@ -331,9 +339,10 @@ def parse_row(self, worksheet, row_index, row):
 		# This is only needed for full events (not the archive sheet),
 		# so only do it if we had a tags column in the first place.
 		if 'tags' in row_dict:
+			shift_tag = common.shifts.calculate_shift(row_dict['event_start'], self.current_shifts)
```

Comment:
typo: The variable is called `self.latest_shifts` above but `self.current_shifts` here.
```suggestion
			shift_tag = common.shifts.calculate_shift(row_dict['event_start'], self.latest_shifts)
```